### PR TITLE
[kubefed] helm3 release from upstream with cert-manager TEMPORARY 

### PR DESCRIPTION
**Pull request by alejandroEsc**
**What type of PR is this?**

Chore

**What this PR does/ why we need it**:

This is essentially a copy of kubefed master branch to be used for getting Kommander helm3 up to speed. We will update Kommander to use upstream kubefed as soon as there is a release.

**Which issue(s) this PR fixes**:

https://jira.d2iq.com/browse/D2IQ-71007

**Special notes for your reviewer**:
HELM 2 Tests will NOT PASS we will merge in regardless. 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Checklist**

* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

File: staging/kubefed/Chart.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
+apiVersion: v2
 description: KubeFed helm chart
 name: kubefed
-version: 0.0.4
-appVersion: v0.1.0-rc6
-home: https://github.com/kubernetes-sigs/kubefed
-maintainers:
-  - name: s12chung
-  - name: alejandroEsc
+version: 0.0.5
```
Comment (samvantran) [inline staging/kubefed/Chart.yaml:4]: Think you need to package and add to `gh-pages`. 0.0.5 is missing https://github.com/mesosphere/charts/tree/gh-pages/staging

which is probably why tests are failing.

Comment (alejandroEsc) [reply staging/kubefed/Chart.yaml:4]: naw, its a requirements field issue, fixing now. Also helm2 will fail on purpose.

**Comment by alejandroEsc** [comment]
Helm2 test not passing is to be expected.

**Comment by alejandroEsc** [comment]
closing since we are getting a kubefed release.

**Pull request closed without merging**